staging/src/k8s.io/sample-controller/controller_test.go

```diff
@@ -17,7 +17,9 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"fmt"
+	"k8s.io/klog/v2/ktesting"
```

Comment:
This import belongs into middle import block.

Reply:
Fixed.